src/cls/IPM/Storage/Module.cls

```diff
@@ -273,7 +300,9 @@ ClassMethod ExecutePhases(pModuleName As %String, pPhases As %List, pIsComplete
 		Set tLifecycle.PhaseList = tPhases
 		Set tPointer = 0
 		Set tFullStart = $zh
+        Set tCustomPhases = tModule.GetCustomPhases()
 		While $ListNext(tPhases,tPointer,tOnePhase) {
+            Set tIsCustomPhase = $ListFind(tCustomPhases, tOnePhase)
```

Comment:
Nit: Rather than converting to a list and doing $ListFind, can leave custom phases in a subscripted array. This list will never be long but seems odd to convert to a list just to use O(n) operation to find entries when a subscript check has better performance anyway.